go-controller/pkg/ovn/controller/services/services_controller_test.go

```diff
@@ -157,10 +156,13 @@ func TestSyncServices(t *testing.T) {
 				},
 			},
 			initialDb: []libovsdbtest.TestData{
-				nodeLogicalSwitch(nodeA),
-				nodeLogicalSwitch(nodeB),
-				nodeLogicalRouter(nodeA),
-				nodeLogicalRouter(nodeB),
+				nodeLogicalSwitch(nodeA, []string{types.ClusterLBGroupName, types.ClusterSwitchLBGroupName}),
```

Comment:
NIT: how about making a const variable `initialLsGrp := ]string{types.ClusterLBGroupName, types.ClusterSwitchLBGroupName})` and using it all over this file?

Reply:
Done.